src/commands/cont.rs

```diff
@@ -36,7 +36,7 @@ impl ContinueCommand {
                 let picked_time_entry = entities
                     .time_entries
                     .iter()
-                    .find(|te| te.id == picked_id)
+                    .find(|te| te.id == picked_id.id)
```

Comment:
```suggestion
                    .find(|te| te.id == picked_entry.id)
```
I guess it makes sense to change the `picked_id` variable to `picked_item` or `picked_entry`